source/reference/api/start.txt

```diff
@@ -204,6 +216,22 @@ Request Body Parameters
 
        .. versionadded:: 1.1
 
+   * - ``verificationMode``
```

Comment:
I think we need to make it clear that it has limitations and either write them here or link to the limitations

Reply:
Fixed